lib/models/Book.js

```diff
@@ -0,0 +1,58 @@
+const pool = require('../utils/pool');
+
+class Book {
+  id;
+  title;
+  released;
+
+  constructor(row) {
+    this.id = row.id;
+    this.title = row.title;
+    this.released = row.released;
+  }
+
+  static async getAll() {
+    const { rows } = await pool.query(
+      'SELECT * FROM books'
+    );
+    return rows.map((row) => new Book(row));
+  }
+
+  async getAuthors() {
+    const { rows } = await pool.query(
+      `SELECT authors.*
+      FROM authors
```

Comment:
the coalesce and json agg would've been helpful for you to return your books a json array.